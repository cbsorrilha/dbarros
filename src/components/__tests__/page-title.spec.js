import React from "react"
import renderer from "react-test-renderer"
import PageTitle from '../page-title';


describe('Snapshot testing the PageTitle', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<PageTitle {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
