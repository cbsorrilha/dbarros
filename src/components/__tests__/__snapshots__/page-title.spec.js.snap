// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the PageTitle renders correctly 1`] = `
<h1
  className="sc-AykKC kQdxEM"
/>
`;
